### ISSUE TITLE
Hide Jupyter extension kernel specific toolbar icons

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#6548]
Hide some or all of the following icons when user selects a non-jupyter kernel (a controller not contributed by jupyter extension)

* [ ] Export
* [ ] Interrupt
* [ ] Restart
* [ ] Variables

@claudiaregio @greazer /cc

**Comment by joyceerhl** [comment]
Dupe of https://github.com/microsoft/vscode-jupyter/issues/6543?

**Comment by claudiaregio** [comment]
@joyceerhl yes this is a dupe, however let's close mine because @DonJayamanne's has check boxes :p

**Comment by greazer** [comment]
Would be best if core contributed some context keys that we can make use of for this.

**Comment by DonJayamanne** [comment]
We can make the changes by adding a global context variable but the UX will be horrible.
Updating Context variabes is async,hence the toolbar icons will either be visible for a fraction of a second & then go away or vice versa.

Basicaly things will appear/disappear when toggling between notebooks.
We had this same UX issue when adding a trusted notebook icon & it was horrible.

We need a different solution or an api from vscode core (to set a context variable when an extension owns the current controller in a notebook)

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/6543
